core/backend/src/test/misc/GeoServices.test.ts

```diff
@@ -840,26 +848,33 @@ describe("GeoServices", () => {
     };
 
     it("should get all CRS", async () => {
-      await validateCRSList(11975, validationRange);
+      await validateCRSList({expectedCount: 12_332, allowedRange: validationRange, includeWorld: true});
     });
 
+    it("should get all CRS, excluding world crs", async () => {
+      await validateCRSList({expectedCount: 11_975, allowedRange: validationRange});
+    })
+
     it("should return CRS that are in the specified range (1)", async () => {
       const extent: Range2dProps = { low: { x: 60.1, y: 61.2 }, high: { x: 62.3, y: 63.4 } };
-      await validateCRSList(82, validationRangeSmall, extent);
+      await validateCRSList({expectedCount: 82, allowedRange: validationRangeSmall, extent});
+      await validateCRSList({expectedCount: 439, allowedRange: validationRangeSmall, extent, includeWorld: true});
     });
 
     it("should return CRS that are in the specified range (2)", async () => {
       const extent: Range2dProps = { low: { x: 0, y: 2 }, high: { x: 1, y: 3 } };
-      await validateCRSList(67, validationRangeSmall, extent);
+      await validateCRSList({expectedCount: 67, allowedRange: validationRangeSmall, extent});
+      await validateCRSList({expectedCount: 424, allowedRange: validationRangeSmall, extent, includeWorld: true});
     });
 
     it("should return CRS that are in the specified range (3)", async () => {
       const extent: Range2dProps = { low: { x: 0.3, y: 2.4 }, high: { x: 1.6, y: 3.77 } };
-      await validateCRSList(62, validationRangeSmall, extent);
+      await validateCRSList({expectedCount: 62, allowedRange: validationRangeSmall, extent});
+      await validateCRSList({expectedCount: 422, allowedRange: validationRangeSmall, extent, includeWorld: true});
```

Comment:
Why do the tests at line 861 and 867 have 357 new systems, while this test has 360 new systems? Shouldn't the number of world systems be the same, no matter the allowed range?

Reply:
That's probably a mistake, checking...

Reply:
The numbers are correct...

Reply:
Any idea why this one has 360 extras while the other two only have 357? It might indicate that something is wrong somewhere, or there could be some other logical explanation.

Reply:
Ah good - I read the debugger wrong, it is 419: 
<img width="141" alt="image" src="https://github.com/user-attachments/assets/9f9dba24-b316-4fae-8918-46ac130bf7c3" />

Phew :)